examples/data-objects/vltava/src/fluidObjects/tabs/tabs.tsx

```diff
@@ -44,7 +45,11 @@ export class TabsFluidObject extends DataObject implements IFluidHTMLView {
     }
 
     protected async hasInitialized() {
-        const registry = await this.context.containerRuntime.IFluidDataStoreRegistry.get("internalRegistry");
+        // TODO: This code should not rely on container globals (i.e. IContainerRuntime)
+        // It should be refactored to pass dependencies in.
```

Comment:
To ease the 0.29 integration, can you add a note similar to this in the breaking that affected code should be refactored to pass the dependencies?  Otherwise I'd expect some extra questions down the road about if scenario is unsupported/discouraged/etc

Reply:
Ah nvm missed the comment about this in the description.